### ISSUE TITLE
New rubygem release

### ISSUE THREAD

**Issue by mdurn** [jkrall/analytical#82]
I was wondering if there could be a new rubygem release since there have been a good amount of changes since the last release. I'm fine using the commit sha for now, but it would be handy.

Thanks!

**Comment by jackphelps** [comment]
At least another version bump; I just spent a really long time banging my head against the wall with the old gem because the repo didn't show any version changes; didn't realize how much it had changed.

**Comment by jkrall** [comment]
Yep, sorry guys.  I'll try to push a new one out by end-of-week.  (I'm swamped today and travelling all day tomorrow).  Maybe one of the other collaborators will have time between now & then.

I'm hoping to take a more active role in maintaining this gem in the future.  For the past year+ its been hard to find time because of my work situation, but that's changing a bit.

**Comment by jackphelps** [comment]
for the record, it seems really useful, I appreciate the work

**Comment by sfsekaran** [comment]
Perhaps I can take a look soon.

**Comment by fabn** [comment]
+1 for this

**Comment by mcnasby** [comment]
Would also love a version bump.  Starting to use this gem more and plan on developing a couple additional modules for it -- already have a Google Tag Manager module in the works.

**Comment by mandrews** [comment]
+1 for new release

**Comment by nj** [comment]
+1 as latest official release was on January 24th, 2012 !

**Comment by jkrall** [comment]
I haven't been using this in production in quite a while (though many other people are, I believe)... can anyone chime in on the stability of the current master?

Happy to cut a new release, but I don't want to break anything for anyone.  I guess I could bump the minor version at least.  Thoughts?

**Comment by infertux** [comment]
:+1: for a new release @jkrall 

I haven't used the version from `master` much so I can't comment on the stability of it.

However, the README on GitHub is full of examples which don't work with the old version from 2012, which is confusing for new users of the Gem.

If you don't know whether there has been changes that will break backward compatibility, may I suggest you push a new major version to Rubygems.org, like `4.0.0`? That way, people will be careful about updating and you won't break anything for anyone as per semantic versioning. I'm sure a lot of people would appreciate not having to use the Git version of the Gem :smile:

**Comment by mdurn** [comment]
I'm only one commit behind (so I don't have google universal fix: https://github.com/jkrall/analytical/commit/e13e361c8d1c2ab371071d58d7833c4f6c7e4c11). I've been using it for Mixpanel and it has been working fine for me. The current gem release had an old version of the mixpanel js and I needed the new one. So people could potentially have an issue if they are using the old js library anywhere.
